packages/types/api/collections.d.ts

```diff
@@ -21,7 +21,6 @@ export interface PartialCollectionRecord {
   ignoreFilesConfigForDiscovery?: boolean,
   name?: string,
   process?: string,
-  reportToEms?: boolean,
```

Comment:
Is this correct?      Dynamo/API collection records *can* still have reportToEms, we're just explicitly dropping the key in postgres writes/migrations/etc as part of this PR.

Reply:
Looks good 👍🏻  675abff831e017953435e1b5db8e059f795983e3